docs/scos/dev/feature-integration-guides/202307.0/glue-api/dynamic-data-api/dynamic-data-api-integration.md

```diff
@@ -358,6 +461,42 @@ class GlueBackendApiApplicationDependencyProvider extends SprykerGlueBackendApiA
 ```
 </details>
 
+<details open>
+<summary markdown='span'>src/Pyz/Zed/Installer/InstallerDependencyProvider.php</summary>
+
+```php
+<?php
+
+namespace Pyz\Zed\Installer;
+
+use Spryker\Zed\DynamicEntity\Communication\Plugin\Installer\DynamicEntityInstallerPlugin;
+use Spryker\Zed\Installer\InstallerDependencyProvider as SprykerInstallerDependencyProvider;
+
+class InstallerDependencyProvider extends SprykerInstallerDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Zed\Installer\Dependency\Plugin\InstallerPluginInterface|\Spryker\Zed\InstallerExtension\Dependency\Plugin\InstallerPluginInterface>
+     */
+    public function getInstallerPlugins(): array
+    {	    
+        return [
+            new DynamicEntityInstallerPlugin(),
+        ];
+    }	    
+}
+```
+</details>
+
+{% info_block warningBox %}
+
+Run the following command after enabling `DynamicEntityInstallerPlugin`:
```

Comment:
Avoid using "run the following command" to introduce code. Instead, focus on what the command does. For example, "When `DynamicEntityInstallerPlugin` is enabled, initiate a database:"
```suggestion
When `DynamicEntityInstallerPlugin` is enabled, run the following command:
```